env/production/kustomization.yaml

```diff
@@ -597,4 +597,11 @@ vars:
       name: application-config
       apiVersion: v1
     fieldref:
-      fieldpath: data.FF_EMAIL_DAILY_LIMIT
\ No newline at end of file
+      fieldpath: data.FF_EMAIL_DAILY_LIMIT
+  - name: SRE_CLIENT_SECRET
+    objref:
+      kind: ConfigMap
+      name: application-config
+      apiVersion: v1
+    fieldref:
+      fieldpath: data.FF_EMAIL_DAILY_LIMIT
```

Comment:
@ben851 is it ok not to provide a value for the `.env` in production just yet?  will that matter when this merges?

Reply:
Oh nevermind the system yelled at me and told me "No."